slackbridge/bots.py

```diff
@@ -15,9 +16,9 @@ class BridgeBot(IRCBot):
     nickname = 'slack-bridge'
 
     def __init__(self, sc, nickserv_pw, slack_uid, channels, user_bots):
+        self.sc = sc
```

Comment:
I do think changing this to `slack_client` or something would be good, as @matthew-mcallister mentioned earlier, but that's my fault for picking a bad name, so it's something that would be good to do in a separate pull request sometime.